### ISSUE TITLE
All settings lost after windows update

### ISSUE THREAD

**Issue by tomstachura** [digitalcreations/MaxTo#286]
After installing the latest windows update today, all maxto settings have been lost (startup on windows start, all key assignments, all region definitions.)

**Comment by vegardlarsen** [comment]
Are you running MaxTo 2.0.0-beta.3? Which version of Windows are you on now? Is the contents of `%AppData%\MaxTo` intact?

Does the file `%AppData%\MaxTo\maxto.log` contain anything special?

**Comment by tomstachura** [comment]
Yes - it seems to update itself. 
W10 - 1803
I can't tell if the data folder was intact. I am sure MaxTo would've recreated all the files. There are files there now.
[maxto.log](https://github.com/digitalcreations/MaxTo/files/2599972/maxto.log)
BTW: Issue 278 (crash when changing shortcuts) is closed, even though the problem is still there.

**Comment by vegardlarsen** [comment]
The log file goes back to the 14th, so it appears your settings wasn't removed. Which specific settings were gone?

From the log file I see that what looks like all the shortcut keys fail as already registered, can it perhaps be that there are two versions of MaxTo running on your machine? (Doesn't really match with the logs though)

**Comment by tomstachura** [comment]
Tej ones I mentioned. Key shortcuts , regions, auto start.

**Comment by vegardlarsen** [comment]
I have seen this happen in someone elses log (privately, through the support mail) as well tonight. I don't know what causes this, but I am looking into the issue. I suspect this may be a installer issue, but I am not sure.

**Comment by vegardlarsen** [comment]
An update here; I am not sure; but I believe some of the crashes you have in your log file will be fixed in beta.4. These are the ones I can remember of the top of my head:

- `HotkeyAlreadyRegisteredException` #280 
- `System.IO.IOException: All pipe instances are busy.` #295

Some of the errors in your log are effects of previous errors (basically because the core process crashed, the communication channel disappeared).

So to sum this up; I am not sure that I have fixed your problem in beta.4, but I assume I have. Please make sure you test when beta.4 is available, and let me know if you are still having problems, and we can re-open this issue.
